test/integration/default_elements.cc

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2017 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <iostream>
+#include <string>
+#include <gtest/gtest.h>
+
+#include "sdf/SDFImpl.hh"
+#include "sdf/parser.hh"
+#include "sdf/Frame.hh"
+#include "sdf/Model.hh"
+#include "sdf/Root.hh"
+#include "sdf/World.hh"
+#include "sdf/Filesystem.hh"
+#include "test_config.h"
+
+//////////////////////////////////////////////////
+TEST(ExplicitlySetInFile, EmptyRoadSphCoords)
+{
+  const std::string test_file =
+    sdf::filesystem::append(PROJECT_SOURCE_PATH, "test", "sdf",
+        "empty_road_sph_coords.sdf");
+
+  sdf::Root root;
+  sdf::Errors errors = root.Load(test_file);
+  EXPECT_TRUE(errors.empty());
+
+  sdf::ElementPtr elementPtr = root.Element();
```

Comment:
Nit: can we name this and the ones below more specifically to `rootElem`, `worldElem` and `roadElem` instead of `elementPtr`? This will help with following errors in test cases

Reply:
True, done: https://github.com/osrf/sdformat/commit/64f226fb26cfae7810fbb5114b29cb1ad8f47279

Reply:
https://github.com/osrf/sdformat/pull/542/commits/761f7bceac18543226bd0fba598fefc06c646f42 instead